tests/spark/test_spark_model_export.py

```diff
@@ -25,13 +25,16 @@
 from mlflow import spark as sparkm
 from mlflow.exceptions import MlflowException
 from mlflow.models import Model
-from mlflow.tracking.artifact_utils import _get_model_log_dir
```

Comment:
are we able to deprecate `_get_model_log_dir` after this PR?

Reply:
I'll follow up with a PR that removes this method and converts all invocations to `_download_artifact_from_uri`.